src/javascript/track.js

```diff
@@ -50,7 +127,7 @@ export default class DropTrack {
 
     // note update
     for (let i = 0; i < this.noteArr.length; ++i) {
-      this.noteArr[i].update();
+      this.noteArr[i].update(i === 0 && this.isUserHoldingNote);
```

Comment:
Recommended that you pull this out into a variable or something.

Reply:
Sure